README.md

```diff
@@ -5,9 +5,14 @@ A quiz that allows people to test how well their perception of people's online i
 
 Before cloning the application, install the Gulp CLI globally by running `npm install gulp-cli -g`. If you're using nvm you can run `nvm use` to run the version of Node this project currently supports. Otherwise refer to the `package.json` file in the root and install and switch to the correct version of Node.
 
+You also need mongodb. Assuming you have homebrew, run `brew install mongo`.
```

Comment:
I would also recommend linking to the mongodb Mac OSX install instructions here because installing mongo also requires creating a `data/db` directory .

Reply:
Good point. On it.